scripts/fuzzing.sh

```diff
@@ -3,16 +3,32 @@ set -e
 source ./scripts/test_lib.sh
 
 GO_CMD="go"
-fuzz_time=${FUZZ_TIME:-"300s"}
-target_path=${TARGET_PATH:-"./server/etcdserver/api/v3rpc"}
-TARGETS="FuzzTxnRangeRequest  FuzzTxnPutRequest  FuzzTxnDeleteRangeRequest"
+fuzz_time=${FUZZ_TIME:-"3s"}
```

Comment:
Why do you change the default value (300s)? I am not sure what's the best practice, but "3s" seems too small.

Reply:
That should not have been changed. Thanks for catching that!